lib/omniauth/strategies/facebook.rb

```diff
@@ -14,6 +14,7 @@ class NoAuthorizationCodeError < StandardError; end
       option :client_options, {
         site: 'https://graph.facebook.com/v2.6',
         authorize_url: "https://www.facebook.com/v2.6/dialog/oauth",
+        auth_scheme: "request_body",
```

Comment:
Maybe better to set it to a symbol like the oauth2 gem was doing before?

Reply:
`auth_scheme: :request_body`